src/popup/locales/en.json

```diff
@@ -116,7 +116,7 @@
     },
     "payloadInfo": {
       "title": "Payload message",
-      "msg": "A payload is an optional public message composed by the sender and sent to the recipient along with a blockchain transaction.<br /><br />Payload can be utilized as additional transaction identifier, destination tag (required by some centralized exchanges when depositing funds), or just to provide any extra information in free-form text."
+      "msg": "A payload is an optional public message composed by the sender and sent to the recipient along with a blockchain transaction.{'<'}br /{'>'}{'<'}br /{'>'}Payload can be utilized as additional transaction identifier, destination tag (required by some centralized exchanges when depositing funds), or just to provide any extra information in free-form text."
```

Comment:
Why the square brackets are wrapped now? This makes the code to look really bad. Maybe there is a different solution for this?

Reply:
when using `@intlify/vue-i18n-loader` html is not allowed in translation texts. When trying to build the extension we will get an error